lib/iris/tests/unit/util/test_equalise_attributes.py

```diff
@@ -16,7 +16,7 @@
 import numpy as np
```

Comment:
The doc string in the file header still references "experimental".  I think it should be iris.util.equalise_attributes:

"""
Unit tests for the :func:`iris.experimental.equalise_cubes.equalise_attributes`
function.
"""